packages/web/src/scss/components/UNSTABLE_Footer/README.md

```diff
@@ -0,0 +1,204 @@
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+        </ul>
+      </nav>
+
+      <nav aria-labelledby="footer-part-four" class="mb-600 mb-tablet-800">
+        <h3 class="typography-body-large-text-bold mb-600" id="footer-part-four">Section headline</h3>
+
+        <ul class="Stack Stack--hasSpacing" style="--stack-spacing: var(--spirit-space-500)">
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+
+    <hr class="UNSTABLE_Divider mb-700 mb-tablet-400" />
+
+    <ul class="UNSTABLE_Footer__copyright">
+      <li>
+        <a href="www.example.com" class="link-secondary">Legal notice</a>
+      </li>
+      <li>
+        <a href="www.example.com" class="link-secondary">Terms of service</a>
+      </li>
+      <li>
+        <a href="www.example.com" class="link-secondary">Privacy policy</a>
+      </li>
+      <li>
+        <a href="www.example.com" class="link-secondary">Manage cookies</a>
+      </li>
+    </ul>
+  </div>
+</footer>
+```
+
+### Footer Columns Variation
+
+In case you need a specific number of columns for each row in the Footer Columns section with links,
+you can use separated `Grid` components. The number of columns can be set for each breakpoint separately.
+
+```html
+<footer class="UNSTABLE_Footer">
+  <div class="Container">
+    <div class="Grid Grid--cols-1 Grid--tablet--cols-3 Grid--desktop--cols-6 mb-tablet-600 mb-desktop-400">
+      <nav aria-labelledby="footer-with-a-lot-of-blocks-part-one" class="mb-600 mb-tablet-400 mb-desktop-800">
+        <h3 class="typography-body-large-text-bold mb-600" id="footer-with-a-lot-of-blocks-part-one">
+          Section headline
+        </h3>
+
+        <ul class="Stack Stack--hasSpacing" style="--stack-spacing: var(--spirit-space-500)">
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+        </ul>
+      </nav>
+
+      <nav aria-labelledby="footer-with-a-lot-of-blocks-part-two" class="mb-600 mb-tablet-400 mb-desktop-800">…</nav>
+      …
+    </div>
+
+    <div class="Grid Grid--cols-1 Grid--tablet--cols-3">
+      <nav aria-labelledby="footer-with-a-lot-of-blocks-part-seven" class="mb-600 mb-tablet-800">…</nav>
+
+      <nav aria-labelledby="footer-with-a-lot-of-blocks-part-eight" class="mb-600 mb-tablet-800">…</nav>
+    </div>
+  </div>
+</footer>
+```
+
+## Color Variants
+
+Currently, Footer comes in two color variants: **cover** (for light
```

Comment:
maybe reword this, because I would expect there is also a Footer--cover variant now.

Reply:
Something like:

> If you need to place the Footer on a dark background, add `Footer--inverted` modifier class to the root element.